### DISCUSSION TITLE
Expression Pedals are missing in Bank Settings UI.

### DISCUSSION THREAD

**Discussion by simonaglover95** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#83] [Editor App Feature Requests] [answered]
When looking at Expression Messages in Bank Settings, only Exp 1A is shown. 

<img width="1596" alt="Screen Shot 2022-08-31 at 10 26 18 am" src="https://user-images.githubusercontent.com/59382082/187599009-0956f734-4798-47ea-9d76-9bd02dba5764.png">

**Comment by konnsim** [comment, marked as answer]
Caused by an API issue:
https://github.com/Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking/issues/84
